src/main/java/tech/fastj/math/Point.java

```diff
@@ -486,8 +486,8 @@ public Point rotate(int angle) {
         float sineOfAngle = (float) Math.sin(angleInRadians);
         float cosineOfAngle = (float) Math.cos(angleInRadians);
 
-        float rotatedX = ((float) x * cosineOfAngle) + ((float) y * sineOfAngle);
-        float rotatedY = ((float) -x * sineOfAngle) + ((float) y * cosineOfAngle);
+        float rotatedX = (x * cosineOfAngle) +  (y * sineOfAngle);
```

Comment:
Looks like you added an extra space there by accident. Mind removing it?

Reply:
Sure Man!